<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CoreEL Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QMainWindow {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0,
                                stop:0 #fff5ec, stop:1 #fdd9b5);
}
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <spacer name="topSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>80</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>360</width>
        <height>380</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>420</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
QFrame {
    background-color: white;
    border-radius: 20px;
    padding: 30px;
    border: 1px solid #f5d4b6;
}
QLabel {
    font: 14pt &quot;Segoe UI&quot;;
    color: #2d2d2d;
    background: transparent;
    border: none;
}
QLineEdit {
    border: 2px solid #f7941d;
    border-radius: 10px;
    padding: 8px;
    font: 11pt &quot;Segoe UI&quot;;
    background-color: #ffffff;
}
QPushButton {
    background-color: #f7941d;
    color: white;
    border-radius: 10px;
    font: bold 12pt &quot;Segoe UI&quot;;
    padding: 8px;
}
QPushButton:hover {
    background-color: #e94e1b;
}
       </string>
      </property>
      <layout class="QVBoxLayout" name="formLayout">
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="labelLogo">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="pixmap">
          <pixmap resource="resources.qrc">:/images/coreel_logo.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="labelTitle">
         <property name="styleSheet">
          <string notr="true">font-size: 18pt; font-weight: bold; margin-bottom: 10px;</string>
         </property>
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_Username">
         <property name="placeholderText">
          <string>Enter Username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_Password">
         <property name="echoMode">
          <enum>QLineEdit::EchoMode::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Enter Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Login">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="bottomSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>80</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>384</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
